_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -62,89 +59,121 @@ sudo su postgres
 psql
 ```
 
-```bash
+```sql
 ALTER USER postgres password 'myStrongPassword';
 ```
 
 Quit psql typing `\q`
 
 Exit postgres user typing `exit`
 
-Navigate to main folder inside postgresql/version/
+Navigate to main folder inside `postgresql/version/`
 ```bash
 cd /etc/postgresql/14/main/
 ```
 We need to edit two files, `pg_hba.conf` and `postgresql.conf`
 ```bash
-sudo nano pg_hba.conf
+sudo nano -w pg_hba.conf
 ```
 Scroll down the file and Add `host, all, all, 0.0.0.0/0, md5`, has to be the first line before `local, all, postgres, , peer`
 
 | TYPE | DATABASE | USER | ADDRESS | METHOD |
 | ---- | -------- | ---- | ------- | ------ |
 | host | all | all | 0.0.0.0/0 | md5 |
 | local | all | postgres |  | peer |
+{: .docs_table}
 
+Close the file and save the changes with `Control+X` and `Y`
 
 ```bash
-sudo nano postgresql.conf
+sudo nano -w postgresql.conf
 ```
 Search for `#listen_addresses='localhost'`, uncomment the line and replace `localhost` for `*`
 
+Close the file and save the changes with `Control+X` and `Y`
+
 Restart the PostgreSQL server
 ```bash
 sudo service postgresql restart
 ```
 
 ### Setup Parse Server
 
-Create a directory
-```bash
-cd ~
-```
+Create a folder called parse-server
 ```bash
-mkdir parse-server
+cd ~ && mkdir parse-server && cd parse-server
 ```
+
+Install Parse Server Locally
 ```bash
-cd parse-server
+sudo yarn add parse-server
 ```
 
-Run the bash script and follow the instructions, the script have some visual issues and the keys generation doesn't work.
-```bash
-sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
```

Comment:
Are we abandoning the bash script?

Reply:
It offers little or no functionality and I decided to install parse-server from npm in this guide (don't remember right now if the bootstrap installs the npm version or the node one..), some functionalities didn't work, and had some visuals faults,  I was looking to refactor the bootstrap into a more dynamic one with more options, etc (I think using node installer or something like that, the same I saw in other products), but not sure If I have the knowledge to do it...